src/cascadia/TerminalApp/userDefaults.json

```diff
@@ -1,30 +1,42 @@
-// This file was generated by %PRODUCT% %VERSION%
+// This file was initially generated by %PRODUCT% %VERSION%
+// It should still be usable in newer versions, but newer versions might have additional
+// settings, help text, or changes that you will not see unless you clear this file
+// and let us generate a new one for you.
 
 // To view the default settings, hold "alt" while clicking on the "Settings" button.
 // For documentation on these settings, see: https://aka.ms/terminal-documentation
-
 {
     "$schema": "https://aka.ms/terminal-profiles-schema",
 
     "defaultProfile": "%DEFAULT_PROFILE%",
 
+    // You can add more global application settings here.
+    // To learn more about global settings, visit https://aka.ms/terminal-global-settings
+
+    // If enabled, selections are automatically copied to your clipboard.
+    "copyOnSelect": false,
+
+    // A profile specifies a command to execute paired with information about how it should look and feel.
+    // Each one of them will appear in the 'New Tab' dropdown,
+    //   and can be invoked from the commandline with `wt.exe -p xxx`
+    // To learn more about profiles, visit https://aka.ms/terminal-profile-settings
     "profiles":
     {
         "defaults":
         {
-            // Put settings here that you want to apply to all profiles
+            // Put settings here that you want to apply to all profiles.
         },
         "list":
         [
             {
-                // Make changes here to the powershell.exe profile
+                // Make changes here to the powershell.exe profile.
                 "guid": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
                 "name": "Windows PowerShell",
                 "commandline": "powershell.exe",
                 "hidden": false
             },
             {
-                // Make changes here to the cmd.exe profile
+                // Make changes here to the cmd.exe profile.
                 "guid": "{0caa0dad-35be-5f56-a8ff-afceeeaa6101}",
                 "name": "cmd",

```

Comment:
This will need merging-in with the cmd.exe localization change.

Reply:
I love it when git is smart enough to handle the merge properly haha. Double checked too 😀